src/ProcessManagerBundle/Controller/ProcessController.php

```diff
@@ -112,6 +112,22 @@ public function clearAction(Request $request): JsonResponse
         $connection = Db::get();
         $connection->executeStatement('DELETE FROM process_manager_processes  WHERE started < UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL ? SECOND))', [$seconds]);
 
+        $logDirectory = \Pimcore::getContainer()->getParameter('process_manager.log_directory');
```

Comment:
please use `$this->getParameter` for this.

Reply:
`$this->container->getParameter()` needs to be used.